packages/api-electron/src/preload/app.ts

```diff
@@ -1,5 +1,14 @@
+const {
+  remote
+} = require('electron');
+const electronApp = remote.app;
```

Comment:
If we only need one import here, might be better just to have:
`const electronApp = require('electron').remote.app`